apps/server/src/modules/files-storage/files-storage.module.ts

```diff
@@ -0,0 +1,61 @@
+import { Module } from '@nestjs/common';
+import { S3Client } from '@aws-sdk/client-s3';
+import { MikroOrmModule } from '@mikro-orm/nestjs';
+import { ALL_ENTITIES } from '@shared/domain';
```

Comment:
We not need  all entities for this module/app.

Reply:
No